packages/common/api.js

```diff
@@ -23,10 +23,16 @@ const callApi = axios.create({
 });
 
 callApi.interceptors.request.use(config => {
-  if (getCookieValue('token')) {
-    const token = `Bearer ${getCookieValue('token')}`;
+  const token = getCookieValue('token');
+  const fingerprint = window.localStorage.getItem(FINGERPRINT);
```

Comment:
You must use the `getTrustedDeviceToken` method with the parameter `generate = false` instead of the direct usage of `localStorage`.

Reply:
why?
We decided to check fingerprint in localStorage when `remember me` checkbox was checked.
For what reasons should we generate it every time instead of just take from localStorage?

Reply:
Replaced ✔️